app/README.md

```diff
@@ -8,59 +8,64 @@ Find the `Domain` and `Client ID` found in the `Settings` tab of your Auth0 Appl
 
 ![auth0nativesettings](https://user-images.githubusercontent.com/35968892/226760563-fbc6e3cf-2d63-4fa1-8fa0-eae98dd80859.png)
 
-Add the Application `Domain` and `Client ID` to the app config file
+Add the Application `Domain` and `Client ID` to the app env file
```

Comment:
For all the steps requiring to add something to the .env file, I think we should delete them, and just add them in the example.env template. Then in the template, that's where we will explain what the environment variable is for. That way, all environment variable documentations are all in just one place, and not all over the READMEs. Additionally, environment variables should be `SCREAMING_SNAKE_CASE_LIKE_THIS` 😄  (i.e. all caps snake case)

Reply:
I like that, it would be a lot more clean. Are we okay with deleting the screen shots then?

Reply:
Yes. As long as we describe it properly in the example.env files.